iOS_SDK/OneSignalSDK/OneSignalCore/Source/OneSignalTrackFirebaseAnalytics.m

```diff
@@ -145,6 +134,10 @@ + (void)trackInfluenceOpenEvent {
     NSString *notificationId = [sharedUserDefaults getSavedStringForKey:ONESIGNAL_FB_LAST_NOTIFICATION_ID_RECEIVED defaultValue:nil];
     NSString *campaign = [sharedUserDefaults getSavedStringForKey:ONESIGNAL_FB_LAST_GAF_CAMPAIGN_RECEIVED defaultValue:nil];
     
+    if (!notificationId || !campaign) {
```

Comment:
Campaigns are mapped to the OneSignal template_id, they are not set a lot of the times, however it is not require by Firebase so we should still allow the event to go through so it isn't skipped completely.

Reply:
If campaign is nil the app will crash so we cannot allow nil

Reply:
https://github.com/OneSignal/OneSignal-iOS-SDK/issues/1253

Reply:
Can we conditionally create the `campaign` key if it is not `nil`?

Reply:
We need to make this fix on main as well

Reply:
agreed, we probably have this same bug